pages/docs/dev-docs/dev-conventions.md

```diff
@@ -20,9 +20,9 @@ summary: This page describes general and coding conventions used in preCICE.
 
 As a rule of thumb again: Orient yourself on the already written code!
 
-There is a settings file for `clang-format`. See the page on \ref tooling for more information on clang-format.
+There is a settings file for `clang-format`. See the page on <a href="https://precice.org/dev-docs-dev-tooling.html#formatting-the-code">Tooling</a> for more information on clang-format.
```

Comment:
Since this is referring to the same website, you can use a relative link:

```suggestion
There is a settings file for `clang-format`. See the page on <a href="dev-docs-dev-tooling.html#formatting-the-code">Tooling</a> for more information on clang-format.
```

The only exception is pages sourced from other repositories (e.g., tutorials), since there we want the link to work also in the respective `README.md` files (which are rendered here).

Reply:
And markdown formatting should work here: `[text](link)`